app/src/pages/Editor/Document.tsx

```diff
@@ -156,6 +156,8 @@ export function Document(): JSX.Element {
       dispatch(goLeft());
     } else if (e.key == 'ArrowRight') {
       dispatch(goRight());
+    } else if (e.key == 'ArrowUp' || e.key == 'ArrowDown') {
```

Comment:
Maybe add an TODO here with reference to https://github.com/audapolis/audapolis/issues/228